v0.1

tranqkhue opened the pull request:
Version 0.1: Basic I/O functions
- Sbus RC Controller
- Roboclaw
- IMU
- GPS

anhquanvgu committed:
Basic teleop function

anhquanvgu committed:
Add usb device name remap rules

tranqkhue committed:
Remove dependent Arduino libraries. Now require to manually update

tranqkhue committed:
Remove dependent Arduino libraries (rosserial). Now require to manually update

tranqkhue committed:
Add install to .gitignore

tranqkhue committed:
Remove ROS' install folder

anhquanvgu committed:
Fork twist_mux: add new functionality: send zero twist when all topics are expired
Update .gitignore and .gitmodules

anhquanvgu committed:
Fork rosserial, and update .gitmodules

anhquanvgu commented on arduino/imu_mag_rosserial/imu_mag_rosserial.ino:
```diff
@@ -0,0 +1,445 @@
+#include "I2Cdev.h"
```
Arduino files concerning imu are no longer needed. Consider to delete them

anhquanvgu commented on src/bugcar_bringup/launch/frames.gv:
```diff
@@ -0,0 +1,18 @@
+digraph G {
```
This not the frames representation for bugcar. Consider to delete

anhquanvgu committed:
Delete settings.json

Part of .vscode. Dont know how it got here

anhquanvgu reviewed (commented):
There are unneeded files: Arduino imu and frames.gv in roslaunch. Consider to delete it

anhquanvgu committed:
Adjust port name for devices in .launch files, change name for sbus .ino code

anhquanvgu committed:
Merge branch 'v0.1' of https://github.com/tranqkhue/bugcar into v0.1

Merge with previous deletion of .vscode components

anhquanvgu committed:
Launch operation:
 - Add diagnostics_agg to view from rqt_robot_monitor
 - Change frame of imu/data topic from 'bno055' to 'base_link'
 - Disable tf publish of roboclaw by renaming
 - Add localization_sensors.launch to launch all sensors related to localization
